### PR TITLE
Intakes

### LINKED ISSUE #477: random website updates
mini updates to the site

### PR DESCRIPTION
**Pull request by nikeshashar**
Closes #477

### CODE REVIEW

File: data/team.yml [hunk 1]
```diff
@@ -56,7 +51,7 @@
 - name: Ben
   image: ben.jpg
   job_title: Coachineer
-  about: Ben has been developing software for over 15 years and has designed and built production systems for large businesses using C#, Java and Ruby.  Ben is an expert in object oriented design and is passionate about clean code.
+  about: Ben has been developing software for over 15 years and has designed and built production systems for large businesses using C#, Java and Ruby. Ben is an expert in object oriented design and is passionate about clean code.
```
Comment (pitchinvasion) [inline data/team.yml]: object-oriented design

File: data/team.yml [hunk 2]
```diff
@@ -91,4 +86,4 @@
 - name: Ana
   image: ana.jpg
   job_title: Coachineer
-  about: Ana has been with us since the very beginning. From onboarding students to tracking down receipts, she is responsible for keeping Makers HQ running! Her official title is Mother Goose and that says it all.
+  about: Ana has been with us since the very beginning. From onboarding students to tracking down receipts, she was responsible for keeping Makers HQ running! She has now jumped ship across to the coaching team after doing the course herself.
```
Comment (pitchinvasion) [inline data/team.yml]: coaching/engineering team

**Comment by pitchinvasion** [comment]
:shipit: once you've fixed those minor changes

File: data/team.yml (hunk 2 above)
Comment (nikeshashar) [inline data/team.yml]: coachineering team?

Comment (pitchinvasion) [inline data/team.yml]: I guess but no one knows what that means apart from us

Comment (nikeshashar) [inline data/team.yml]: No. That debate should be another ticket. Can we close this one so it's consistent with the current site?

Comment (pitchinvasion) [inline data/team.yml]: Sure

**Pull request merged**